README.md

```diff
@@ -45,29 +46,26 @@ Using [ember-block-slots](https://github.com/ciena-blueplanet/ember-block-slots)
     }}
   {{/block-slot}}
   {{#block-slot slot 'header'}}
-    <div class="primary-title">Test title</div>
+    <div class="primary-title">TEST TITLE</div>
   {{/block-slot}}
   {{#block-slot slot 'body'}}
     <div class="custom-body">
       <div>Test Information block</div>
+      <div>Test Information block</div>
     </div>
   {{/block-slot}}
-  {{#block-slot slot 'cancel'}}
-    {{frost-button
-      autofocus=true
-      size='medium'
-      priority='tertiary'
-      text='Cancel'
-    }}
-  {{/block-slot}}
-  {{#block-slot slot 'confirm'}}
-    {{frost-button
-      size='medium'
-      priority='primary'
-      text='Confirm'
-    }}
+  {{#block-slot slot 'footer' as |controls|}}
+    {{controls.cancel
+      text='Cancel'}}
```

Comment:
Can we show this using the defaulted text and mention the defaults (and text prop) in the description above?